### PR TITLE
fix: navigate to main page if navigated directly

### PR DESCRIPTION
**Pull request by omBratteng**
## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

AS-728

### Preview domain
https://as-728-fix-back-button.preview.app.daily.dev

### CODE REVIEW

File: packages/webapp/components/layouts/PlusLayout/PlusHeader.tsx [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
   Button,
   ButtonVariant,
 } from '@dailydotdev/shared/src/components/buttons/Button';
 import { ArrowIcon } from '@dailydotdev/shared/src/components/icons';
 import { useRouter } from 'next/router';
+import { webappUrl } from '@dailydotdev/shared/src/lib/constants';
 
 export const PlusHeader = (): ReactElement => {
-  const { back } = useRouter();
+  const { back, replace } = useRouter();
```
Comment (rebelchris) [inline packages/webapp/components/layouts/PlusLayout/PlusHeader.tsx:12]: Technically could be something non daily dev right?
Maybe we should somehow check if jt was external?

Comment (rebelchris) [reply packages/webapp/components/layouts/PlusLayout/PlusHeader.tsx:12]: I guess for now it's fine maybe.
More of a nice to have

**Review by rebelchris — APPROVED** [review]
approving for now

File: packages/webapp/components/layouts/PlusLayout/PlusHeader.tsx (hunk 1 above)
Comment (omBratteng) [reply packages/webapp/components/layouts/PlusLayout/PlusHeader.tsx:12]: > I guess for now it's fine maybe. More of a nice to have

Yeah, would be nice to have. But probably need to add a global state to track our internal previous page

Comment (omBratteng) [reply packages/webapp/components/layouts/PlusLayout/PlusHeader.tsx:12]: Testing https://as-728-fix-back-button.preview.app.daily.dev/plus

Comment (omBratteng) [reply packages/webapp/components/layouts/PlusLayout/PlusHeader.tsx:12]: Testing2 https://as-728-fix-back-button.preview.app.daily.dev/

**Pull request merged**